packages/pluggableWidgets/barcode-scanner-web/src/hooks/useCodeScanner.tsx

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useState } from "react";
-import { BarcodeFormat, BrowserMultiFormatReader, DecodeHintType } from "@zxing/library";
+import { BarcodeFormat, BrowserMultiFormatReader, DecodeHintType } from "@zxing/library/cjs";
```

Comment:
Why is this change necessary?

Reply:
Unfortunately on windows it fails drastically if not using it. @leonardomendix needs to test it to see the impact, but should not be a problem.